includes/templates/template_default/templates/tpl_account_history_info_default.php

```diff
@@ -130,39 +130,61 @@
  */
 if (!empty($order->statuses)) {
 ?>
-
 <h2 id="orderHistoryStatus"><?php echo HEADING_ORDER_HISTORY; ?></h2>
 <table id="myAccountOrdersStatus">
     <tr class="tableHeading">
         <th scope="col" id="myAccountStatusDate"><?php echo TABLE_HEADING_STATUS_DATE; ?></th>
         <th scope="col" id="myAccountStatus"><?php echo TABLE_HEADING_STATUS_ORDER_STATUS; ?></th>
+<?php
+    $extra_headings = [];
+    $zco_notifier->notify('NOTIFY_ACCOUNT_HISTORY_INFO_OSH_HEADINGS', ['order' => $order], $extra_headings);
```

Comment:
Instead of passing an array of `['order' => $order]` why not just pass `$order` directly? Easier to use directly without having to de-reference an array on the receiving end.